dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -121,6 +121,9 @@
     <Import Project="..\Slider\Slider.vcxitems" Label="Shared" />
     <Import Project="..\TimePicker\TimePicker.vcxitems" Label="Shared" />
     <Import Project="..\ToolTip\ToolTip.vcxitems" Label="Shared" />
+    <Import Project="..\FlipView\FlipView.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
```

Comment:
I wasn't sure what was and wasn't supposed to have this check. If it should be on all the templates you could add to the ones I added right above this.

Reply:
Hmm, this was generated by the script, and i guess it depends on whether the store app is using that specific control or not.

I think I'll remove the condition since the controls we added are not really new controls and it might be used by store app.